pkgs/development/compilers/rust/rustc.nix

```
@@ -10,6 +10,7 @@
 , version
 , sha256
 , patches ? []
+, fast_cross
```

```suggestion
, fastCross
```

Otherwise it's inconsistent with every other option / variable.